### PR TITLE
[JENKINS-31776] CLI command online-node extracted from core to CLI

### PR DESCRIPTION
**Pull request by pjanouse**
Please review. Thx, Pavel

### CODE REVIEW

File: core/src/main/java/hudson/cli/OnlineNodeCommand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    protected int run() throws Exception {
+
+        boolean errorOccurred = false;
+
+        final Jenkins jenkins = Jenkins.getInstance();
+
+        Computer computer = jenkins.getComputer(computerName);
+
+        if (computer == null) {
+            stderr.println(hudson.model.Messages.Computer_NoSuchSlaveExists(computerName, null));
```
Comment (pjanouse) [inline core/src/main/java/hudson/cli/OnlineNodeCommand.java]: Note this is only a temporary solution. After merge JENKINS-31417, the better message (NoSuchSlaveExistsWithoutAdvice) would be available.

File: core/src/main/java/hudson/cli/OnlineNodeCommand.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2013 Red Hat, Inc.
```
Comment (olivergondza) [inline core/src/main/java/hudson/cli/OnlineNodeCommand.java]: 2015

Comment (pjanouse) [inline core/src/main/java/hudson/cli/OnlineNodeCommand.java]: Fixed

**Comment by oleg-nenashev** [comment]
:+1:

**Pull request merged**